### ISSUE TITLE
Expansion indicator triangle not clickable

### ISSUE THREAD

**Issue by hotsphink** [glandium/tabstats#10]
If you want to expand a section, you can hover over one of the triangle twisty things and it'll light up, but clicking on it does nothing. Clicking on the label works. Both should work.

**Comment by euank** [comment]
Note that it's the use of css to fake the list bullet styling that does this ultimately. That hack is here: https://github.com/glandium/tabstats/blob/v0.1.6/abouttabs.css#L92-L95

**Comment by glandium** [comment]
I reverted that hack from dc287de5bc6f8a20cada9075c0fc65ba1be2a998 in 18b5d4f, but that didn't fix this.

**Comment by euank** [comment]
Indeed, I misremembered what the solution was (moving the handler from span -> li instead).

I investigated and almost fixed this once before, which is why I felt confident in my assertion.. even though it turned out I totally misremembered the details
